css/index.css

```diff
@@ -104,14 +118,45 @@ p {
 .middle-content .boxes .box {
     width: 12.5%;
     height: 100px;
-    background: black;
+    /* background: black; */
```

Comment:
Great job on adding a second class to each box div in order to change the styling. Just note that the reason you had to comment out this line in order for the background color of the boxes to change color is due to specificity. `.middle-content .boxes .box` is more specific than `.boxes .box1`. 

You can achieve the effect of changing the backgrounds without commenting out this line by using `.middle-content .boxes .box1`